src/WorkflowCore/WorkflowCore.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
     <PackageReference Include="System.Threading.Thread" Version="4.3.0" />
     <PackageReference Include="System.Linq.Queryable" Version="4.3.0" />
+    <PackageReference Include="OpenTelemetry.Api" Version="1.2.0-rc2" />
```

Comment:
Taking a dependency on pre-release packages could be a blocker for some organizations.

Reply:
downgraded to last stable version